lib/prefix.c

```diff
@@ -1114,6 +1114,15 @@ void prefix_free_lists(void *arg)
 	prefix_free(&p);
 }
 
+void prefix_flowspec_allocate(struct prefix *p, uint8_t *pnt, int size)
```

Comment:
is this intended to be called from multiple places - that's what the commit headline implies/

Reply:
yes. public API.
actually, it serves only bgp flowspec.
any recommendation?